### ISSUE TITLE
Filters

### ISSUE THREAD

**Issue by mvarnagiris** [mvarnagiris/financius#20]
Ability to filter out information in certain screens by everything :)

**Comment by frandavid100** [comment]
Is it similar to the current search feature in Categories?

**Comment by frandavid100** [comment]
What I would like is to have a single Filter screen that can be reached by pressing the magnifying glass from any of the main screens and shows entries from all types (categories, accounts, people, transactions, groups and so on).

![financius filter](https://f.cloud.github.com/assets/694794/1512434/bc4436f6-4aca-11e3-9151-cb50fe5693e8.png)

Doing it like this would save the user from having to navigate to the right screen before starting the search (e.g. he wouldn't have to press the overflow → settings → categories → magnifying glass), and entering as little as three characters should filter out unwanted results anyway.

**Comment by mvarnagiris** [comment]
It's not search. It's filter. Create new issue "Global search" with these screenshots. This might be a good idea

**Comment by frandavid100** [comment]
Done.

**Comment by firnasz** [comment]
I don't know whether my suggestion is proposed already in this topic.

When it comes to large amount of transactions like at least 3 months of data, it will be difficult to dig through old data, because we have to scroll down a lot.

It is very useful if we have an option to customize the transactions screen such as filter to see what we want for example,

show current month transactions
show last month transactions
show transactions of a specific category or subcategory
show transactions within a specific date range
show transaction of a specific account

If this is already implemented, may I know how to do this?

**Comment by firnasz** [comment]
When will this feature be implemented? I eagerly waiting

**Comment by MaybeThisIsRu** [comment]
Eagerly waiting for this particular feature. :)

**Comment by ibnumalik** [comment]
This will be the most useful features if it is implemented ;)
